edu.harvard.i2b2.xml/xsd/cell/ont_1.1/ONT_QRY.xsd

```diff
@@ -103,6 +103,7 @@
         <xs:sequence>
             <xs:element name="parent" type="xs:string"/>
         </xs:sequence>
+        <xs:attribute name="numLevel" type="xs:int" default="1" />
```

Comment:
We think the default should be -1 so it's not executed if numLevel isn't present?

Reply:
then what would you pass in if you want all levels

Reply:
Ok, so numLevel<1 is all levels? If numLevel isn't present, it needs to run the original code path though please.